cli/cmd/policy.go

```diff
@@ -0,0 +1,202 @@
+		RunE: createPolicy,
+	}
+)
+
+func init() {
+	// add the lql command
+	rootCmd.AddCommand(policyCmd)
+
+	// add sub-commands to the lql command
+	policyCmd.AddCommand(policyCreateCmd)
+
+	// run specific flags
+	setPolicySourceFlags(policyCreateCmd)
+}
+
+func setPolicySourceFlags(cmds ...*cobra.Command) {
+	for _, cmd := range cmds {
+		if cmd == nil {
+			return
+		}
+		action := strings.Split(cmd.Use, " ")[0]
+
+		// file flag to specify a policy from disk
+		cmd.Flags().StringVarP(
+			&policyCmdState.File,
+			"file", "f", "",
+			fmt.Sprintf("path to a policy to %s", action),
+		)
+		/* repo flag to specify a policy from repo
+		cmd.Flags().BoolVarP(
+			&policyCmdState.Repo,
+			"repo", "r", false,
+			fmt.Sprintf("id of a policy to %s via active repo", action),
+		)*/
+		// url flag to specify a policy from url
+		cmd.Flags().StringVarP(
+			&policyCmdState.URL,
+			"url", "u", "",
+			fmt.Sprintf("url to a policy to %s", action),
+		)
+	}
+}
+
+// for commands that take a policy as input
+func inputPolicy(cmd *cobra.Command, args []string) (string, error) {
+	// if running via repo
+	if policyCmdState.Repo {
+		return inputPolicyFromRepo()
+	}
+	// if running via file
+	if policyCmdState.File != "" {
+		return inputPolicyFromFile(policyCmdState.File)
+	}
+	// if running via URL
+	if policyCmdState.URL != "" {
+		return inputPolicyFromURL(policyCmdState.URL)
+	}
+	// if running via editor
+	action := strings.Split(cmd.Use, " ")[0]
+	return inputPolicyFromEditor(action)
+}
+
+func inputPolicyFromRepo() (policy string, err error) {
+	err = errors.New("NotImplementedError")
+	return
+}
+
+func inputPolicyFromFile(filePath string) (policy string, err error) {
+	fileData, err := ioutil.ReadFile(filePath)
+
+	if err != nil {
+		err = errors.Wrap(err, "unable to read file")
+		return
+	}
+
+	policy = string(fileData)
+	return
+}
+
+func inputPolicyFromURL(url string) (policy string, err error) {
+	msg := "unable to access URL"
+
+	response, err := http.Get(url)
+	if err != nil {
+		err = errors.Wrap(err, msg)
+		return
+	}
+	defer response.Body.Close()
+
+	if response.StatusCode != 200 {
+		err = errors.Wrap(errors.New(response.Status), msg)
+		return
+	}
+
+	body, err := ioutil.ReadAll(response.Body)
+	if err != nil {
+		err = errors.Wrap(err, msg)
+		return
+	}
+	policy = string(body)
+	return
+}
+
+func inputPolicyFromEditor(action string) (policy string, err error) {
+	prompt := &survey.Editor{
+		Message:  fmt.Sprintf("Type a policy to %s", action),
+		FileName: "policy*.json",
+	}
+	err = survey.AskOne(prompt, &policy)
+
+	return
+}
+
+func createPolicy(cmd *cobra.Command, args []string) error {
+	policy, err := inputPolicy(cmd, args)
+	if err != nil {
+		return errors.Wrap(err, "unable to create policy")
+	}
+
+	cli.Log.Debugw("creating policy", "policy", policy)
+
+	create, err := cli.LwApi.Policy.CreatePolicy(policy)
+
+	if err != nil {
+		return errors.Wrap(err, "unable to create policy")
+	}
+	if cli.JSONOutput() {
```

Comment:
This is a good point to use a new line if you want, mainly since you are now doing a different process by checking for JSON output or Human-readable:
```suggestion

	if cli.JSONOutput() {
```

Reply:
Fixed.